src/components/Menudetails.css

```diff
@@ -0,0 +1,25 @@
+.tabs {
```

Comment:
AFAIK, we don't need to create a seperate .css file as we are using tailwind css.